src/index.html

```diff
@@ -15,8 +15,104 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconect"
+      href="https://fonts.google.com/specimen/Roboto"
```

Comment:
There is a typo in the `rel` attribute value, it should be "preconnect" instead of "preconect".